src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.h

```diff
@@ -82,9 +82,10 @@ class ChaseMovementGenerator : public TargetedMovementGeneratorMedium<T, ChaseMo
 
         static void _clearUnitStateMove(T* u) { u->ClearUnitState(UNIT_STATE_CHASE_MOVE); }
         static void _addUnitStateMove(T* u)  { u->AddUnitState(UNIT_STATE_CHASE_MOVE); }
-        bool EnableWalking() const { return false;}
+        bool EnableWalking(T*) const { return false; }
         bool _lostTarget(T* u) const { return u->GetVictim() != this->GetTarget(); }
         void _reachTarget(T*);
+        void _updateSpeed(T* owner) { }
```

Comment:
OK, of course. I see your 3.3.5 branch is 21 commits behind, maybe it could have something to do with it. Anyway, thanks for working on this. :)